ngraph/test/frontend/paddlepaddle/op/assign_value.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
As discussed, these tests are not compiled, have build issues and are failed. Probably should be removed

Reply:
already removed these in 0f9058cb8edab2b1cd0b740e8717ac6582eca6a3